recipes/noaa-nclimgrid/recipe.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+from apache_beam import Create
+
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern
+from pangeo_forge_recipes.transforms import OpenURLWithFSSpec, OpenWithXarray, StoreToZarr
+
+input_url_pattern = 's3://noaa-nclimgrid-daily-pds/v1-0-0/grids/{yyyy}/ncdd-{yyyymm}-grd-scaled.nc'
+
+
+def format_function(time):
+    return input_url_pattern.format(yyyy=time.strftime('%Y'), yyyymm=time.strftime('%Y%m'))
+
+
+dates = pd.date_range('1951-01-01', '2023-09-01', freq='MS')
+concat_dim = ConcatDim(name='time', keys=dates, nitems_per_file=None)
+pattern = FilePattern(format_function, concat_dim)
+
+recipe = (
+    Create(pattern.items())
+    | OpenURLWithFSSpec()
+    | OpenWithXarray(file_type=pattern.file_type)
+    | StoreToZarr(
```

Comment:
@sbquinlan In the context of these AWS datasets we want to be creating Kerchunk indexes rather than Zarr archives so we should be using the [WriteCombinedRecipes](https://pangeo-forge.readthedocs.io/en/latest/composition/styles.html#open-with-kerchunk-write-to-virtual-zarr) rather than `StoreToZarr`.